google/cloud/spanner/interval.h

```diff
@@ -128,6 +131,19 @@ inline Interval operator/(Interval lhs, double rhs) { return lhs /= rhs; }
  */
 StatusOr<Interval> MakeInterval(absl::string_view);
 
+/**
+ * Add the Interval to the Timestamp in the civil-time space defined by
+ * the TimeZone. Saturates the Timestamp result upon overflow.
+ */
+Timestamp Add(Timestamp const&, Interval const&, absl::TimeZone);
```

Comment:
This is adding `absl::TimeZone` to the public API. In general, we try to avoid `absl::[A-Z].*` in the public API because Abseil does not make any guarantees about backwards compatibility for these types.